index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="container">
+
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form id="survey-form" action="">
+    
+
+      
+      <div  class="option-column">
+        <label for="name" id="name-label">Name</label>
+        <input placeholder="Your Name" required id="name" type="text">
+        <label for="email" id="email-label">Email</label>
+        <input placeholder="Your Email" required id="email" type="email">
+        <label for="number" id="number-label">Number</label>
+        <input max="10" min="2" placeholder="Your Number" id="number" type="number">
+      </div>
```

Comment:
Please try to use a "fieldset" tag instead a general "div". That makes more readable for web browsers 🔢